auctions/views.py

```diff
@@ -30,11 +30,7 @@ class BidStatusView(UserPassesTestMixin, LoginRequiredMixin, TemplateView):
     template_name = "addon/bidders.html"
 
     def test_func(self):
-        if self.request.user.is_active:
-            if not self.request.user.stripe_customer:
-                self.template_name = "addon/register_card.html"
```

Comment:
This removes the last instance of `register_card.html`, so we should remove the file too.